hw/ip_templates/gpio/dv/env/seq_lib/gpio_rand_straps_vseq.sv.tpl

```diff
@@ -27,73 +26,68 @@ class ${module_instance_name}_rand_straps_vseq extends ${module_instance_name}_b
     super.new(name);
   endfunction
 
-  // Read hw_straps_data_in and hw_straps_data_in_valid and
+  // Read hw_straps_data_in and hw_straps_data_in_valid
```

Comment:
No big deal, but I'd probably "wrap less" for things like this. The standard setup is 100 columns.

Reply:
This is not fixed. Please could you undo some of the wrapping, so that the lines go out to something more like 100 columns?